controllers/rhmi/rhmi_controller.go

```diff
@@ -434,6 +436,81 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	return retryRequeue, err
 }
 
+func (r *RHMIReconciler) createSilence(installation *rhmiv1alpha1.RHMI) error {
+
+	host := "localhost:9093"
+	endpoint := "/api/v1/silences"
+
+	alertName := "KeycloakInstanceNotAvailable"
```

Comment:
I noticed we are silencing only this one alert^
And the JIRA is talking about more general problem - `Please suppress alerts that are caused by a cluster uninstall`. No other alerts can fire during uninstallation?

Reply:
The other alerts within Keycloak are warning alerts and as far as Im aware shouldnt page SRE.
 All other integreatly alerts are removed as part of uninstall https://github.com/integr8ly/integreatly-operator/blob/master/controllers/rhmi/rhmi_controller.go#L499:L510

Reply:
Cool, thanks for explanation.
So this means that any critical alerts created by other operators need to be silenced during uninstall?
I know that we plan to allow 3scale-operator create the alerts - [MGDAPI-428](https://issues.redhat.com/browse/MGDAPI-428), should we expand this JIRA to make sure those alerts become silenced on uninstall?

Reply:
updated to allow additional alerts to be added. it should only be a matter of adding the names of critical alerts to the `alertsToSilence` array.